src/garage/sampler/worker.py

```diff
@@ -0,0 +1,155 @@
+"""Worker class used in all Samplers."""
+import abc
+from collections import defaultdict
+
+import numpy as np
+
+from garage.experiment import deterministic
+
+
+class Worker(abc.ABC):
+    """Worker class used in all Samplers."""
+
+    def __init__(self, seed, max_path_length, worker_number):
```

Comment:
this is a nit, but the order i would probably expect in this API for `worker_number` to come first.

+1 making seed required.  I think that this is a good practice for reproducibility because it forces the implementer to at least choose a seed to pass (instead of silent repro bugs because of a missing seed).

Reply:
I'm just gonna make them require being passed by keyword. They're all integers, so they're easy to mix up.